packages/rspack/src/NormalModule.ts

```diff
@@ -63,8 +63,8 @@ const deprecateAllProperties = <O extends object>(
 	return newObj;
 };
 
-export class NormalModule {
```

Comment:
🤨 In the fact, I don't like this rule, `NormalModule` seems useful as a type before but it is forced to use like `typeof NormalModule` now and I think it will be a little different from webpack

so it's an api change without benefits, what do you think

Reply:
I think we don't have to follow the recommended rules for now when there is no benefit. 🤔

Reply:
If you agree, I will close this PR. 😌